resource_packs/files/unobtrusive/no_fog/biomes_client.json

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
Fix the error and the pack is also not at all working
![Screenshot_20241014_015650_Minecraft](https://github.com/user-attachments/assets/50e341fd-4a95-49e2-9002-508762bbbc0c)
![Screenshot_20241014_015722_Minecraft](https://github.com/user-attachments/assets/b5313f21-c8f9-47f7-87be-3df373c049e8)

Reply:
how did you find these errors
I didnt even know you couldnt change the fog distance over 1 when using render (render means : fog distance * render distance) will look into this :)

Reply:
> how did you find these errors I didnt even know you couldnt change the fog distance over 1 when using render (render means : fog distance * render distance) will look into this :)

I just loaded in game and these errors were thrown in content log gui and the pack was not working

Reply:
Still can’t replicate the errors